charts/connectors/README.md

```diff
@@ -10,7 +10,7 @@ This page describes the official Redpanda Connectors Helm Chart. In particular,
 For instructions on how to install and use the chart, including how to override and customize the chart’s values, refer to the [deployment documentation](https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/kubernetes/k-deploy-connectors/).
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.13.1](https://github.com/norwoodj/helm-docs/releases/v1.13.1)
+Autogenerated from chart metadata using [helm-docs v1.14.2](https://github.com/norwoodj/helm-docs/releases/v1.14.2)
```

Comment:
Super nit: This should be part of the previous commit as that's the commit to introduced this change.

Reply:
Ok I will change that

Reply:
It will more difficult to review, but I'm ok to squash it.